lib/ansible/modules/database/postgresql/postgresql_privs.py

```diff
@@ -386,6 +386,16 @@ def get_all_sequences_in_schema(self, schema):
         self.cursor.execute(query, (schema,))
         return [t[0] for t in self.cursor.fetchall()]
 
+    def get_all_functions_in_schema(self, schema):
+        if not self.schema_exists(schema):
+            raise Error('Schema "%s" does not exist.' % schema)
+        query = """SELECT format('%I(%s)', p.proname, oidvectortypes(p.proargtypes))
```

Comment:
Hello, first of all, please, change it to something like this:
```
query = ("SELECT ..."
              "...."
              "..." % ...)
```
imho, it looks better and more effective.
This PR looks good. I should try it manually.